tests/test_app.py

```diff
@@ -1,25 +1,54 @@
 import pytest
-from app import Game
 
+from app import TestTypingSpeed
 
-def test_user_can_choose_between_english_or_romanian():
-    game = Game()
-    choice = "english"
-    assert game.choose_language(choice) == "ENGLISH Language Selected."
 
-def test_an_english_sentence_is_shown_in_console():
-    game = Game()
-    choice = "english"
-    with open("english_sentences.txt", "r") as file:
-        content = file.readlines()
-        content = [phrase.strip() for phrase in content]
-    assert game.get_random_sentence(choice) in content
+@pytest.fixture
+def game():
+    return TestTypingSpeed()
 
-def test_an_romanian_sentence_is_shown_in_console():
-    game = Game()
-    choice = "romanian"
-    with open("romanian_sentences.txt", "r") as file:
-        content = file.readlines()
-        content = [phrase.strip() for phrase in content]
-    assert game.get_random_sentence(choice) in content
 
+@pytest.mark.parametrize(
+    "language_choice, expected_language_selected",
+    [
+        ("english", "ENGLISH Language Selected",),
+        ("romanian", "ROMANIAN Language Selected"),
+        ("asdhasiuhdasihduas", "INVALID option")
+    ]
+)
+def test_chosen_language(game, language_choice, expected_language_selected):
```

Comment:
The choose_language method expects input from the user, does the test stop to get user input? If so find a way that it doesn't. The tests need to run in isolation.